src/precompiles.rs

```diff
@@ -42,11 +56,156 @@ pub fn istanbul_precompiles(
         7 => todo!(), // TODO: implement alt_bn128_mul()
         8 => todo!(), // TODO: implement alt_bn128_pair()
         9 => todo!(), // TODO: implement blake2f()
+        EXIT_TO_NEAR_ID => {
+            exit_to_near(&input, &context);
+            Some(Ok((ExitSucceed::Returned, Vec::new(), 0)))
+        }
+        EXIT_TO_ETHEREUM_ID => {
+            exit_to_ethereum(&input, &context);
+            Some(Ok((ExitSucceed::Returned, Vec::new(), 0)))
+        }
         // Not supported.
         _ => None,
     }
 }
 
+#[allow(dead_code)]
+fn exit_to_near(input: &[u8], context: &Context) {
+    // TODO: Determine the correct amount of gas
```

Comment:
Note TODO

Reply:
I'm solving all TODO(#51) before merging, will mark as resolved this comment though the TODO itself is not resolved.